tests/unit-tests.js

```diff
@@ -51,6 +51,23 @@ function assertThrows(message, func) {
   }
 })();
 
+
+(function tagsAreNotBenchmarkNames() {
+  const benchmarkNames = new Set(BENCHMARKS.map(e => e.name));
+  for (const benchmark of benchmarks) {
+    for (const tag of benchmark.tags) {
+      assertFalse(benchmarkNames.has(tag), `'${tag}' is also a benchmark name`);
+    }
+  }
+})();
+
+(function tagsHasWasmOrJS() {
+  for (const benchmark of benchmarks) {
+    const tags = benchmark.tags;
+    assertTrue(tags.has("wasm") || tags.has("js"), `'${benchmark.name}' has no 'js' or 'wasm' tag`);
```

Comment:
Should this also test for Xor? I guess a test could arguably be both though.

Reply:
initially thought about xor this but we definitely want mixed workloads at some point too with wasm-gc.